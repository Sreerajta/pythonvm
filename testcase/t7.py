print "simple function"
def funtest():
 print "entered function"
 
print "calling function:"

funtest()
