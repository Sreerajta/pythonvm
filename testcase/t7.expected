simple function
calling function:
entered function
